build.gradle

```diff
@@ -569,19 +569,23 @@ subprojects {
                                 "-javacoptions:-source 8",
                                 "-javacoptions:-target 8"
                                 ]
+                            if (project.name.is('checker')) {
+                                args += [
+                                        "-javacoptions:-Xbootclasspath/p:${projectDir}/dist/jdk8.jar",
+                                ]
+                            }
+                        } else {
+                            args += [
+                                    "-javacoptions:--add-modules jdk.jdeps",
```

Comment:
Could you add a comment why this `jdk.jdeps` export is needed only here?
Why doesn't CheckerMain also need this?

Reply:
I added a comment above this line.  `checker/jtreg/nullness/defaultsPersist/ReferenceInfoUtil.java` uses `jdk.deps` module.  CheckerMain doesn't need it because the Checker Framework doesn't use it.